doc/design/subscription-config.md

```diff
@@ -133,3 +133,23 @@ spec:
         memory: "128Mi"
         cpu: "500m"
 ```
+
+### NodeSelector
+
+The `nodeSelector` field defines a [NodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) for the Pod created by OLM.
```

Comment:
nit: this is a bit vague. Could we say "for the pods associated with the operator deployment" instead?

This also future-proofs it a bit where in the future OLM supports more than just deploying operators, something we discussed recently

Reply:
/SGTM

Reply:
Technically this config is applied to all deployments created by OLM. OLM also creates deployments for webhooks, which could be stored in the same deployment as the operator but isn't guaranteed, so I don't know how OLM could only create deployments for operators.